src/store/version.ts

```diff
@@ -9,7 +9,7 @@ import { loadVersion, saveVersion } from "./local-storage";
 export function stateIsInvalid() {
     const localVersion = loadVersion();
     if (localVersion) {
-        if (isIncompatibleVersion(localVersion)) {
+        if (isIncompatibleVersion(localVersion) || !loadState()) {
```

Comment:
I hope you tested this thoroughly! This bit took a bit of time to get right when I wrote it.

Reply:
If local storage's RootState cannot be loaded, it returns undefined. So if loadState() is falsy, then whatever is stored in local storage cannot be loaded, either because there is nothing there, or the state there cannot be mapped to RootState. In either case it should be considered an invalid state.